Development/src/pages/subscriptions.js

```diff
@@ -31,164 +31,146 @@ import JsonIcon from '../components/JsonIcon';
 
 const cookies = new Cookies();
 
-export class SubscriptionsList extends React.Component {
-    constructor(props) {
-        super(props);
-        this.filter = '';
-        this.filterObject = {};
-        this.state = {
-            data: [],
-        };
-        this.nextPage = this.nextPage.bind(this);
-        this.setFilter = this.setFilter.bind(this);
-        this.filterPage = this.filterPage.bind(this);
-        this.firstLoad = this.firstLoad.bind(this);
-        this.firstLoad();
-    }
-
-    async firstLoad() {
+export const SubscriptionsList = () => {
+    const firstLoad = async () => {
         const params = {
             filter: {},
             pagination: { page: 1, perPage: 10 },
-            sort: { field: 'timestamp', order: 'DESC' },
```

Comment:
Don't think `pagination` should be here either as it is only relevant to offset-limit pagination, and the NMOS APIs don't support sorting either, so no `sort` either.